api/app/auto_spatial_advisory/nats_consumer.py

```diff
@@ -26,9 +28,13 @@ def parse_nats_message(msg: Msg):
     if msg.subject == sfms_file_subject:
         decoded_msg = json.loads(json.loads(msg.data.decode()))
         run_type = RunType.from_str(decoded_msg['run_type'])
-        run_date = datetime.datetime.strptime(decoded_msg['run_date'], "%Y-%m-%d").date()
-        for_date = datetime.datetime.strptime(decoded_msg['for_date'], "%Y-%m-%d").date()
-        return (run_type, run_date, for_date)
+        run_date = datetime.strptime(decoded_msg['run_date'], "%Y-%m-%d").date()
+        for_date = datetime.strptime(decoded_msg['for_date'], "%Y-%m-%d").date()
+
+        # SFMS doesn't give us a timezone, but from the 2022 data it runs in local time
+        # so we localize it as such then convert it to UTC
+        run_datetime = get_utc_datetime(datetime.fromisoformat(decoded_msg['create_time']))
+        return (run_type, run_date, run_datetime, for_date)
```

Comment:
is it necessary to have both `run_date` AND `run_datetime`? Why not just datetime?

Reply:
Theoretically yes, I just want to keep both to verify any discrepancy in real SFMS data and it's easier to drop data after the fact.